### PR TITLE
Allow shortcuts with mod key to be fired inside kbar

### PR DESCRIPTION
**Pull request by thecoorum**
Resolves https://github.com/timc1/kbar/issues/243

### CODE REVIEW

**Comment by notmedia** [comment]
@timc1 Hi, could you review this PR please?

**Comment by timc1** [comment]
Hey @notmedia, apologies for the late response here!

Regarding this spec, I'm not quite sure I understand what's going on – when I test this implementation, every single command opens up kbar (ie hit `g+d` and we navigate to the docs page as well as toggle kbar). What is this accomplishing? 🤔

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 node_modules
-lib
```
Comment (timc1) [inline .gitignore:2]: Any particular reason this is being removed?

File: example/src/hooks/useThemeActions.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export default function useThemeActions() {
       name: "Change theme…",
       keywords: "interface color dark light",
       section: "Preferences",
+      shortcut: ["$mod+1"]
```
Comment (timc1) [inline example/src/hooks/useThemeActions.tsx:34]: Hitting mod+1 will trigger a browser tab switch as this event default is not prevented.

**Comment by notmedia** [comment]
Hi, @timc1! Thank you for bringing attention 🤝

My case in #243 was smoothly simple - make ability to toggle some shortcut (not all, only that programmaticly allowed) when an input element is focused.

You can see my project to prevent miss understanding - https://github.com/getezy/ezy.
Here I have text-input (code editor) where the user most often focusing on it. But he can't use some shortcuts except default `$mod+K` because of this function https://github.com/timc1/kbar/blob/main/src/utils.ts#L101.

I understand why shortcuts blocked when inputs are focused - because of key sequence when user for example typing something.
So, my proposal was to make shortcuts configurable which may work in this situation or like @thecoorum said we can stop ignoring all `$mod + ..` sequences. In first implementation of this PR it was done, but now I don't understand what going on too.

**Comment by timc1** [comment]
Let's try to just work with enabling `$mod` keys as a valid key?

**Comment by notmedia** [comment]
Yeah, good solution

**Comment by thecoorum** [comment]
Closing this one as it need more attention and some more edge cases need to be covered

**Pull request closed without merging**